gamemodes/terrortown/gamemode/client/cl_shopeditor.lua

```diff
@@ -16,6 +16,12 @@ local net = net
 local pairs = pairs
 local IsValid = IsValid
 
+ShopEditor = ShopEditor or {}
```

Comment:
I think this should be `shopeditor` instead

Reply:
Might be logical client vs server, but the complete cl_shopeditor uses ShopEditor, so why would I use a new global variable?

Reply:
Okay I get it, that you have all modules in small letters. I can rename it once merged and when I clean up the code. This might lead to a huge conflict with the other PR otherwise.